### ISSUE TITLE
Unity version compatibility 

### ISSUE THREAD

**Issue by HeadClot** [sabresaurus/SabreCSG#15]
Hey this is a bit of a question / Request.

1. Would it be possible to get the version of Unity that SabreCSG is compatible with listed at the top of the readme?

2. What version of Unity is SabreCSG compatible with?

Thank you for your time,

HeadClot

**Comment by derekhendrickx** [comment]
@HeadClot I'm using the latest version of Unity (2017.2) and it works fine.

**Comment by Henry00IS** [comment]
I put a note about the Unity version compatibility [in the wiki](https://github.com/sabresaurus/SabreCSG/wiki/FAQ#what-unity-versions-are-supported).

Now that the project is actively maintained again I am assuming people seeing commits of "20 days ago" should be indication enough that the latest Unity versions are supported. I doubt any of us have the time to go through historic versions of Unity to make sure every tiny change is compatible. So I prompted users to just open a new pull request if they encounter something we missed (in case they are unable to update their Unity because of a long term project).

As such I wouldn't want to guarantee and promote any version compatibility dating back too long ago on the front page in the readme. Only people stuck in old Unity versions will now actively seek out this issue and the wiki pages to find out if SabreCSG works for them.

**TL;DR:**
Put a note about version compatibility [in the wiki](https://github.com/sabresaurus/SabreCSG/wiki/FAQ#what-unity-versions-are-supported), closing this issue to clean up the issue browser.
